src/GitHub.VisualStudio.UI/Styles/ActionLinkButton.xaml

```diff
@@ -10,7 +10,7 @@
     <Setter Property="Template">
       <Setter.Value>
         <ControlTemplate TargetType="Button">
-          <TextBlock>
+          <TextBlock TextTrimming="CharacterEllipsis">
```

Comment:
The better way is to have it as a templated setter (like the `Background`, `Focusable` and `Foreground` properties above). The problem there is that the `Button` object doesn't have a `TextTrimming` property (because buttons can have non-text content), so we need to switch from using a button to using a `GitHubActionLink` and add a custom DependencyProperty to it that we can then use to set the trimming.